src/Model.ts

```diff
@@ -346,6 +348,25 @@ export class Model implements IModel {
   }
 
   /**
+   * Compares a regular object to a model.
+   *
+   * @param {object} comparingObject Object that will be compared to the model
+   * @param {IModelIsEqualParams} params Options for comparison
+   * @returns {boolean} Is the object equal to the model
+   * @memberof Model
+   */
+  public isEqual(comparingObject: object, params: IModelIsEqualParams = {ignoreId: true}): boolean {
+    const runParams: IModelIsEqualParams = assign({}, params, {
+      ommitPaths: params.ommitPaths || [TYPE_PROP],
+    });
+    if (runParams.ignoreId) {
+      runParams.ommitPaths.push(this.static.idAttribute);
+    }
+    const nativeObject = omit(this.toJS(), runParams.ommitPaths);
```

Comment:
On second throught... Why even create a whole `runParams` object if you only need an array?